Libplanet/Net/Swarm.cs

```diff
@@ -729,7 +739,7 @@ IEnumerator IEnumerable.GetEnumerator()
 
         private async Task BindingProxies()
         {
-            while (Running)
+            while (Running && !_stopping)
```

Comment:
This predicate seems repeated.  How about changing the definition of `Running` property, or having a more property to return `Running && !_stopping`?

Reply:
Changing `Running` may be bit tricky because other processes that needed while stopping also rely `Running`.

Adding a more property seems to okay, but I didn't catch proper name….

Reply:
How about renaming `Running` to `Started` and adding `Running => Started && !_stopping`?